src/main/resources/db/changelog/changeset/0009_agency_topic/agencyTopicTrigger.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TRIGGER `agencyservice`.`agency_topic_update` BEFORE UPDATE ON `agencyservice`.`agency_topic` FOR EACH ROW BEGIN
```

Comment:
what is now with the trigger? just implement like all others?
but then we have it twice - in my point of view - once updated per trigger in db (not tested, because the h2 don't have) and implemented in the java world...

Reply:
let's keep the trigger to keep it consistent with other existing solutions.